package.json

```diff
@@ -2,7 +2,10 @@
   "name": "home-assistant.io",
   "description": "Home Assistant Website & Documentation",
   "version": "1.0.0",
-  "dependencies": {},
+  "dependencies": {
+    "bundler": "^0.8.0",
+    "gem": "^2.4.3"
```

Comment:
Why are these in here?

Reply:
> Some comments to look at.
> 
> I'm not sure how Netlify would pick up this page? Could you elaborate on that?

Not sure I believe Gem has changed these files. I will change them back